wire-ios-data-model/Tests/MLS/MLSServiceTests.swift

```diff
@@ -2650,4 +2650,143 @@ class MLSServiceTests: ZMConversationTestsBase, MLSServiceDelegate {
         XCTAssertTrue(waitForCustomExpectations(withTimeout: 0.5))
     }
 
+    // MARK: - Proteus to MLS migration
+
+    func test_startProteusToMLSMigration_succeeds() async throws {
```

Comment:
we should think about how to split these tests, we have >2000 lines here, extension?

Reply:
Yep we can probably split it up but even if you create these uses cases I suppose there still some core MLS operations which are shared between them all so the MLS service would go away completely?

Reply:
This can be done in a follow-up PR, right?